src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -189,8 +189,41 @@ void GcsServer::DoStart(const GcsInitData &gcs_init_data) {
       /*ms*/ RayConfig::instance().event_stats_print_interval_ms(),
       "GCSServer.deadline_timer.debug_state_event_stats_print");
 
+  global_gc_throttler_ =
+      std::make_unique<Throttler>(RayConfig::instance().global_gc_min_interval_s() * 1e9);
```

Comment:
Is it right to * 1e9?

Reply:
It should work because `NodeManager` does it the same way.